validation.js

```diff
@@ -1,259 +1,242 @@
 function checkAge(ageVal, varName) {
-  if (!ageVal) throw `Error: You must supply a ${varNmae}.`;
-  if (typeof ageVal !== "number") throw `Error: ${varName} should be a number.`;
-  if (ageVal === NaN) throw `Error: ${varName} age cannot be NaN.`;
-  if (ageVal < 13 || ageVal > 100)
-    throw `Error: ${varName} is out of valid range.`;
-  return ageVal;
+	if (!ageVal) throw `Error: You must supply a ${varNmae}.`;
+	if (typeof ageVal !== 'number') throw `Error: ${varName} should be a number.`;
+	if (ageVal === NaN) throw `Error: ${varName} age cannot be NaN.`;
+	if (ageVal < 13 || ageVal > 100) throw `Error: ${varName} is out of valid range.`;
+	return ageVal;
 }
 
 // Validates bio inputs.
 function checkBio(bioVal, varName) {
-  if (!bioVal) throw `Error: You must supply a ${varName}.`;
-  if (typeof bioVal !== "string") throw `Error: ${varName} should be a string.`;
-  bioVal = bioVal.trim();
-  if (bioVal.length === 0)
-    throw `Error: ${varName} cannot be an empty string or string with just spaces`;
-  if (bioVal.length > 500)
-    throw `Error: ${varName} cannot be longer than 500 characters.`;
-  return bioVal;
+	if (!bioVal) throw `Error: You must supply a ${varName}.`;
+	if (typeof bioVal !== 'string') throw `Error: ${varName} should be a string.`;
+	bioVal = bioVal.trim();
+	if (bioVal.length === 0)
+		throw `Error: ${varName} cannot be an empty string or string with just spaces`;
+	if (bioVal.length > 500) throw `Error: ${varName} cannot be longer than 500 characters.`;
+	return bioVal;
 }
 
 //Validates boolean inputs
 function checkBoolean(boolVal, varName) {
-  if (boolVal === "undefined") throw `Error: You must supply a ${varName}.`;
-  if (typeof boolVal !== "boolean")
-    throw `Error: ${varName} should be a boolean.`;
-  return boolVal;
+	if (boolVal === 'undefined') throw `Error: You must supply a ${varName}.`;
+	if (typeof boolVal !== 'boolean') throw `Error: ${varName} should be a boolean.`;
+	return boolVal;
 }
 
 // Validates comment inputs.
 function checkComment(comment, varName) {
-  if (!comment) throw `Error: You must supply a ${varName}.`;
-  if (typeof comment !== "string")
-    throw `Error: ${varName} should be a string.`;
-  comment = comment.trim();
-  if (comment.length === 0)
-    throw `Error: ${varName} cannot be an empty string or string with just spaces`;
-  if (comment.length > 280)
-    throw `Error: ${varName} cannot be longer than 280 characters.`;
-  return comment;
+	let string_format = /^[A-Za-z !@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?]*$/;
+	if (!comment.match(string_format)) throw `Error: ${varName} must be a valid string!`;
+	if (!comment) throw `Error: You must supply a ${varName}.`;
+	if (typeof comment !== 'string') throw `Error: ${varName} should be a string.`;
+	comment = comment.trim();
+	if (comment.length === 0)
+		throw `Error: ${varName} cannot be an empty string or string with just spaces`;
+	if (comment.length > 280) throw `Error: ${varName} cannot be longer than 280 characters.`;
```

Comment:
```suggestion
	if (comment.length > 280) throw `Error: ${varName} cannot be longer than 280 characters.`;
	if (!isNaN(strVal))
		throw `Error: ${comment} is not a valid value for ${varName} as it only contains digits`;
```

Reply:
@saipranav789 said to allow any input for the comments, so I removed all restrictions.